test/lit.cfg.py

```diff
@@ -223,13 +245,15 @@ def one_line(s):
     ('%test-dir-in', os.path.join('%input-dir', '%test-basename')),
     ('%test-proof-diff-out', os.path.join('%output-dir', '%test-basename.proof.out.diff')),
     ('%test-proof-debug-diff-out', os.path.join('%output-dir', '%test-basename.proof.debug.out.diff')),
+    ('%test-shm-buffer', os.path.join('/', '%test-basename.shm.buffer')),
```

Comment:
Why is this path defined to start with `/`?

Reply:
It is a POSIX requirement that names of shared memory objects start with `/`.

Reply:
Interesting, didn't know that!